src/forest.py

```
@@ -1,6 +1,7 @@
 import os
 import math
 from .common import add_dem_layer, get_main_def
+from .progress_manager import ProgressManager
```

Ruff (0.11.9)

`.progress_manager.ProgressManager` imported but unused

Remove unused import: `.progress_manager.ProgressManager`

(F401)